packages/chaire-lib-frontend/src/components/pages/LoadingPage.tsx

```diff
@@ -7,10 +7,10 @@
 import React from 'react';
 import Loader from 'react-spinners/HashLoader';
 
-const LoadingPage = function (props) {
+const LoadingPage = function (loadingPageProps) {
```

Comment:
Maybe here it's rpeferable to add the correct type instead of silencing the warning by renaming the props. For simplicity, you can inline the type `props: { message?: string }`